docs/components/board/_index.md

```diff
@@ -99,1209 +99,9 @@ import (
 
 The board component supports the following methods:
 
-{{< readfile "/static/include/components/apis/board.md" >}}
+{{< readfile "/static/include/components/apis/generated/board-table.md" >}}
 
-Additionally, the nested `GPIOPin`, `Analog`, and `DigitalInterrupt` interfaces support the following methods:
-
-[`GPIOPin`](#gpiopin-api) API:
-
-{{< readfile "/static/include/components/apis/gpiopin.md" >}}
```

Comment:
Should these files also be getting deleted?

Reply:
Thank you, good catch, removed all three I missed:

```
        deleted:    static/include/components/apis/analog.md
        deleted:    static/include/components/apis/digitalinterrupt.md
        deleted:    static/include/components/apis/gpiopin.md
```